src/Bicep.LangServer.UnitTests/Configuration/BicepConfigChangeHandlerTests.cs

```diff
@@ -236,7 +236,7 @@ private void RefreshCompilationOfSourceFilesInWorkspace(string bicepFileContents
             var workspace = new Workspace();
 
             var configurationManager = new ConfigurationManager(new IOFileSystem());
-            var bicepCompilationManager = new BicepCompilationManager(server, BicepCompilationManagerHelper.CreateEmptyCompilationProvider(configurationManager), workspace, BicepCompilationManagerHelper.CreateMockScheduler().Object, BicepTestConstants.CreateMockTelemetryProvider().Object, new LinterRulesProvider());
+            var bicepCompilationManager = new BicepCompilationManager(server, BicepCompilationManagerHelper.CreateEmptyCompilationProvider(configurationManager), workspace, BicepCompilationManagerHelper.CreateMockScheduler().Object, BicepTestConstants.CreateMockTelemetryProvider().Object, new LinterRulesProvider(), BicepTestConstants.FileResolver);
```

Comment:
[nit]: it would be easier to review and reason through this code if the args here were wrapped, reducing the need to navigate right and lose context of the statements prior.